### ISSUE TITLE
add other nordic services similar to dataverse.no

### ISSUE THREAD

**Issue by wikfeldt** [coderefinery/social-coding#6]
would be good if participants in all nordic countries can get a useful pointer to services they can use

**Comment by annefou** [comment]
In norway, https://archive.norstore.no/ (NIRD archive is also widely used by some communities)

**Comment by wikfeldt** [comment]
here's a (google-translated) answer to a question I sent to a person working at the KTH library:
"""
In Sweden there is the Swedish National Data Service, which so far has mainly made available research data in the social sciences and the humanities, but they have received funding from VR to expand and become a national data service with the ambition to cover all research areas. [removed text] During the year we will evaluate whether KTH will join as a consortium member in SND. Their web: snd.gu.se
 
Then there are also some more subject-specific data repositories / support organizations such as:
ICOS for climate data: http://www.icos-sweden.se/
Bolin center climate / geodata: https://bolin.su.se/data/
NBIS for life science (sequence –omics data) https://nbis.se/infrastructure/
Is there a list of repositories in Sweden here:
https://www.re3data.org/search?query=&countries[]=SWE
"""

**Comment by bast** [comment]
Thank you!

**Comment by rkdarst** [comment]
Here's what I say (Finland):

1. Use a domain-specific repository for your type of data.  If you
work in a field, you probably know what they are, but you can use the
[registery of research data repositories](https://www.re3data.org/) to
search for repositories.  Consider their characteristics, because not
every repository is equally good: does it make your data citeable?
Does it have a long-term plan?

2. Use a general-purpose international repository.  We recommend
Zenodo, for ease of use and public support.

3. Use a national service.  The common ones in Finland are
[IDA](https://ida.fairdata.fi/login) for general data and
[FSD](https://www.fsd.uta.fi/en/) for social data.  These should be
considered first if data won't be open, since they have some options
for non-open data.  For more information on Finnish resources for open
data, see [https://www.fairdata.fi/en/](fairdata.fi).


-----

The first two steps are obviously not Finland-specific.  I (and some
others here) recommend international resources first, for visibility
and also because the larger something is, the longer term it can be
expected to be around and maintained.

**Comment by bast** [comment]
Actually this does not fit so well anymore into this lesson after it has been reorganized in December - but perhaps fits much better into https://coderefinery.github.io/reproducible-research/02-organizing-projects/?

**Comment by annefou** [comment]
Yes and what about having a real section on data management (not only project organisation)?

**Comment by bast** [comment]
I agree, AFAIU our goal is to develop a real stand-alone session on this.

**Comment by bast** [comment]
In Denmark: http://www.opendata.dk (thanks to Max)

**Comment by mehimself** [comment]
We asked our participants in Aalborg with the following results (not all nordic):

https://opendata.dk portal for open data in denmark
https://transparency.entsoe.eu/ (electrical grid data, EU)
https://fred.stlouisfed.org/ (Economic data)
http://brugstedet.dk/ (danish portal for sharing geographical data)
https://data.chhs.ca.gov/ California health and human services open data portal
https://www.gapminder.org/data/ Data from the Gapminder project 
https://www.kaggle.com/ Datasets for data science challenges

**Comment by bast** [comment]
I think this should be in reproducible research and linked to from "Social coding". I am moving the issue.
